post.hbs

```diff
@@ -22,8 +22,9 @@
       </ul>
       {{/post}}
 
-      {{!> disqus}}
+      {{> disqus}}
```

Comment:
And also disqus, like google analytics, must to be enabled directly by the user.